### DISCUSSION TITLE
Camera is not working after update

### DISCUSSION THREAD

**Discussion by svenschedlbauer** [jomjol/AI-on-the-edge-device#1479] [General] [answered]
Hello,
I have installed AI new and it was running, but I was not able to choose digital and analog model. After I made an update I could choose digital and analog model, but now I see no picture in the overview and I can not make a new reference picture.... :-( 
I rebooted, installed new, flashed new, but nothing helps. What could be the solution?
Hope somebody can help me...
Regards, Sven
![Overview](https://user-images.githubusercontent.com/110197932/205493538-82682f80-1d70-46bc-ad19-0b1d68031451.JPG)
![create new reference](https://user-images.githubusercontent.com/110197932/205493539-0757a5ab-2c60-4913-a249-5dde5a35f2bb.JPG)
![actual reference](https://user-images.githubusercontent.com/110197932/205493541-cf253acf-b254-4506-a092-f7d3b854bff1.JPG)

**Comment by jomjol** [comment]
Can you send a serial log? Looks like something is going wrong during the startup.

**Reply by svenschedlbauer** [reply]
This is the log file. I really have no idea, what the problem can be. I already thought it is the camera, but it already worked.

[       4] 2022-12-04T19:08:59	<INF>	[MAIN] =================================================
[       4] 2022-12-04T19:08:59	<INF>	[MAIN] ==================== Startup ====================
[       4] 2022-12-04T19:08:59	<INF>	[MAIN] =================================================
[       4] 2022-12-04T19:08:59	<INF>	[MAIN] Branch: 'HEAD', Revision: 961662f, Date/Time: 2022-12-04 14:50, Web UI: Release: v13.0.4 (Commit: 961662f) 
[       4] 2022-12-04T19:08:59	<INF>	[MAIN] Reset reason: Via esp_restart
[       7] 2022-12-04T19:09:02	<INF>	[MAIN] =================================================
[       7] 2022-12-04T19:09:02	<INF>	[MAIN] ================== Main Started =================
[       7] 2022-12-04T19:09:02	<INF>	[MAIN] =================================================
[       9] 2022-12-04T19:09:04	<INF>	[MAIN] Initialization completed successfully!
[     302] 2022-12-04T19:13:57	<INF>	[OTA] Reboot triggered by Software (5s).
[     302] 2022-12-04T19:13:57	<WRN>	[OTA] Reboot in 5sec
[       4] 2022-12-04T19:14:02	<INF>	[MAIN] =================================================
[       4] 2022-12-04T19:14:02	<INF>	[MAIN] ==================== Startup ====================
[       4] 2022-12-04T19:14:02	<INF>	[MAIN] =================================================
[       4] 2022-12-04T19:14:02	<INF>	[MAIN] Branch: 'HEAD', Revision: 961662f, Date/Time: 2022-12-04 14:50, Web UI: Release: v13.0.4 (Commit: 961662f) 
[       4] 2022-12-04T19:14:02	<INF>	[MAIN] Reset reason: Via esp_restart
[       7] 2022-12-04T19:14:05	<INF>	[MAIN] =================================================
[       7] 2022-12-04T19:14:05	<INF>	[MAIN] ================== Main Started =================
[       7] 2022-12-04T19:14:05	<INF>	[MAIN] =================================================
[       9] 2022-12-04T19:14:07	<INF>	[MAIN] Initialization completed successfully!
[      12] 2022-12-04T19:15:43	<INF>	[SNTP] NTP Status OK
[      12] 2022-12-04T20:15:43	<INF>	[SNTP] Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
[      12] 2022-12-04T20:15:43	<INF>	[MAIN] =================================================
[      12] 2022-12-04T20:15:43	<INF>	[MAIN] ================== Main Started =================
[      12] 2022-12-04T20:15:43	<INF>	[MAIN] =================================================
[      14] 2022-12-04T20:15:45	<INF>	[MAIN] Initialization completed successfully!

**Comment by svenschedlbauer** [comment]
I got it now. It is working. :-) 👍 
But he don't show me an Value in the Overview and under Error I can read
Rate too high - Read: 680.3452 - Pre: 679.0000
What does it mean?

**Comment by jomjol** [comment, marked as answer]
You have the MaxRate value. Now the change to the new value is bigger than the maximal allowed change.
Update the PreValue to the current reading manually and maybe increase the MaxRate Value.
